pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = "CycloneDX-Buildroot"
+version = "1.0.7"
+description = "Create a software bill of materials from the direct dependencies of a Buildroot generated project"
+authors = ["Robert Smigielski <ptdropper@users.noreply.github.com>"]
+license = "Apache-2.0"
+readme = "README.md"
+packages = [
+    { include = "generateBuildrootSBOM" },
+]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+packageurl-python = "^0.14.0"
```

Comment:
id _suggest_ to use `packageurl-python = ">=0.14, <2"`.

According to SemanticVersioning, `^0.14.0` shoudl be equal to `>=0.14.0, <0.15`.
the developers of `packageurl-python` are working in a versing model where they intend to not have breaking changes, but still do not use v1, since they are not feature complete ... 
Which means that `>=0.x, <1` or similar should work well.